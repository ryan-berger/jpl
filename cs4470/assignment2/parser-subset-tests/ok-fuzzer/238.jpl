show X2W.6U.(x)
print "BA)%"
write image O to "+"
let o = D
write image 2428 to "c"
show D(p)
let g7D_ = 08
read image "" to k
read image "mbAx" to C5T_8D_
read image "" to R
show o()
let O = .524320
time write image 77336. to "]D"
read image "Y" to O
