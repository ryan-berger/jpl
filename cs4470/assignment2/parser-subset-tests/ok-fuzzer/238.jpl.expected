(ShowCmd (CallExpr X2W.6U. (VarExpr x)))
(PrintCmd "BA)%")
(WriteImageCmd (VarExpr O) "+")
(StmtCmd (LetStmt (ArgLValue (VarArgument o)) (VarExpr D)))
(WriteImageCmd (IntExpr 2428) "c")
(ShowCmd (CallExpr D (VarExpr p)))
(StmtCmd (LetStmt (ArgLValue (VarArgument g7D_)) (IntExpr 8)))
(ReadImageCmd "" (VarArgument k))
(ReadImageCmd "mbAx" (VarArgument C5T_8D_))
(ReadImageCmd "" (VarArgument R))
(ShowCmd (CallExpr o))
(StmtCmd (LetStmt (ArgLValue (VarArgument O)) (FloatExpr 0)))
(TimeCmd (WriteImageCmd (FloatExpr 77336) "]D"))
(ReadImageCmd "Y" (VarArgument O))
